web/modules/custom/parser/src/Command/ParserCommand.php

```diff
@@ -3,10 +3,11 @@
 namespace Drupal\parser\Command;
 
 use Symfony\Component\Console\Input\InputInterface;
-use Symfony\Component\Console\Input\InputOption;
 use Symfony\Component\Console\Output\OutputInterface;
 use Drupal\Console\Core\Command\Command;
-use Drupal\parser\Handler\ParserHandler;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Drupal\parser\Reader\LocationReader;
+use Drupal\parser\Writer\DB\LocationWriter;
 
 /**
  * Class ParserCommand.
```

Comment:
how can I test this changes?

Reply:
run 'docker-compose run php drupal parser'